tools/ui-components/src/button.css

```diff
@@ -28,3 +30,7 @@
   font-size: 16px;
   padding: 12px 24px;
 }
+
+.fcc-style {
+  @apply bg-darkGreen;
+}
```

Comment:
This is just to validate that tailwind is working. The button does not match our styles and its implementation needs to be updated (IMO). I am planning to tear it down and rebuild it in a coming up PR as the pilot component.